test/extended/security/scc.go

```diff
@@ -339,3 +537,100 @@ func runAsRootPSPSSR() *securityv1.PodSecurityPolicySelfSubjectReview {
 		},
 	}
 }
+
+func createPodAdminRoleOrDie(ctx context.Context, oc *exutil.CLI, sa *corev1.ServiceAccount) {
+	framework.Logf("Creating role")
+	rule := rbacv1helpers.NewRule("create", "update").Groups("").Resources("pods", "pods/exec").RuleOrDie()
+	_, err := oc.AdminKubeClient().RbacV1().Roles(sa.Namespace).Create(
+		ctx,
+		&rbacv1.Role{
+			ObjectMeta: metav1.ObjectMeta{Name: "podadmin"},
+			Rules:      []rbacv1.PolicyRule{rule},
+		},
+		metav1.CreateOptions{},
+	)
+	o.Expect(err).NotTo(o.HaveOccurred())
+
+	framework.Logf("Creating rolebinding")
+	_, err = oc.AdminKubeClient().RbacV1().RoleBindings(sa.Namespace).Create(ctx, &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace:    sa.Namespace,
+			GenerateName: "podadmin-",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind: "ServiceAccount",
+				Name: sa.Name,
+			},
+		},
+		RoleRef: rbacv1.RoleRef{
+			Kind: "Role",
+			Name: "podadmin",
+		},
+	}, metav1.CreateOptions{})
+	o.Expect(err).NotTo(o.HaveOccurred())
+}
+
+func createServiceAccount(ctx context.Context, oc *exutil.CLI, namespace string) *corev1.ServiceAccount {
+	framework.Logf("Creating ServiceAccount")
+	sa, err := oc.AdminKubeClient().CoreV1().ServiceAccounts(namespace).Create(ctx, &corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{GenerateName: "test-sa-"}}, metav1.CreateOptions{})
+	if err != nil {
+		panic(fmt.Errorf("unexpected error: %v", err))
+	}
+
+	framework.Logf("Waiting for ServiceAccount %q to be provisioned...", sa.Name)
+	err = exutil.WaitForServiceAccountWithSecret(oc.AdminKubeClient().CoreV1().ServiceAccounts(namespace), sa.Name)
+	o.Expect(err).NotTo(o.HaveOccurred())
+
+	return sa
+}
+
+func createPodsecuritypolicyselfsubjectreviewsRoleBindingOrDie(ctx context.Context, oc *exutil.CLI, sa *corev1.ServiceAccount) {
+	framework.Logf("Creating pspssr role")
```

Comment:
creating what?

Reply:
createPodSecurityPolicySelfSubjectReviewsRoleBindingOrDie :D

Reply:
my point is - please expand the shortcut in the log, otherwise people will hate you whenever they will be trying to understand what's going on in here :)